charts/tempo-distributed/templates/metrics-generator/service-metrics-generator.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.metricsGenerator.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "tempo.metricsGeneratorFullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "tempo.metricsGeneratorLabels" . | nindent 4 }}
+spec:
+  ports:
+    - name: http
+      port: 3100
```

Comment:
i feel like these ports are likely configurable somewhere. we should probably use that value instead of hardcoding.

Reply:
FYI: I used the ingestor as bases. I could be a good idea to use templates to generator components.

I will update this for metrics generator.